### ISSUE TITLE
Why is the reward low and not increased in grpo training？How to solve？

### ISSUE THREAD

**Issue by AXy1527** [huggingface/open-r1#436]
my config
# Model arguments
model_name_or_path: ../experiment/models/Qwen2.5-1.5B-Instruct
#model_revision: main
torch_dtype: bfloat16
attn_implementation: flash_attention_2

# Data training arguments
dataset_name: ../experiment/datasets/NuminaMath-TIR/data
dataset_configs:
- default
system_prompt: "You are a helpful AI Assistant that provides well-reasoned and detailed responses. You first think about the reasoning process as an internal monologue and then provide the user with the answer. Respond in the following format: <think>\n...\n</think>\n<answer>\n...\n</answer>"
# Num processes is less by 1 as vLLM is using 1 GPU
num_processes: 3

# GRPO trainer config
bf16: true
use_vllm: true
vllm_device: auto
vllm_gpu_memory_utilization: 0.7
do_eval: false
gradient_accumulation_steps: 16
gradient_checkpointing: true
gradient_checkpointing_kwargs:
  use_reentrant: false
#hub_model_id: Qwen2.5-1.5B-Open-R1-GRPO
#hub_strategy: every_save
learning_rate: 2.0e-05
log_completions: true
log_level: info
logging_first_step: true
logging_steps: 5
logging_strategy: steps
lr_scheduler_type: cosine
max_prompt_length: 512
max_completion_length: 1024
max_steps: -1
num_generations: 6
num_train_epochs: 1
output_dir: outputs/Qwen2.5-1.5B-Open-R1-GRPO-no-difficulty
overwrite_output_dir: true
per_device_eval_batch_size: 16
per_device_train_batch_size: 8
push_to_hub: false
report_to:
- none
reward_funcs:
- accuracy
- format
#- tag_count
reward_weights:
- 1.0
- 1.0
#- 1.0
save_strategy: "steps"
save_steps: 100
#save_total_limit: 1
seed: 42
warmup_ratio: 0.1

**Comment by LeonEricsson** [comment]
Could you provide some metric logs?

**Comment by AXy1527** [comment]
step540: {'loss': 0.0175, 'grad_norm': 0.7757256627082825, 'learning_rate': 1.8891780137353036e-05, 'rewards/accuracy_reward': 0.033333334140479565, 'rewards/format_reward': 0.02500000074505806, 'reward': 0.05833333460614085, 'reward_std': 0.1124404639005661, 'completion_length': 557.3916839599609, 'kl': 0.4371826'71875, 'epoch': 0.24}
step545: {'loss': 0.0226, 'grad_norm': 14.432273864746094, 'learning_rate': 1.885621449063547e-05, 'rewards/accuracy_reward': 0.061458335258066654, 'rewards/format_reward': 0.013541667070239783, 'reward': 0.07500000279396772, 'reward_std': 0.11841528080403804, 'completion_length': 538.670849609375, 'kl': 0.5654296075, 'epoch': 0.24}


> Could you provide some metric logs?
